bundle/src/main/java/com/adobe/acs/commons/fam/package-info.java

```diff
@@ -17,5 +17,5 @@
  * limitations under the License.
  * #L%
  */
-@aQute.bnd.annotation.Version("2.2.0")
+@aQute.bnd.annotation.Version("3.0.0")
```

Comment:
I did introduce new FAM behaviors in this update: The task manager and action managers now support a way to halt a running task dead in its tracks.  This changed method signatures but was absolutely necessary to support a stop button.